tests/calculateRank.test.js

```diff
@@ -16,3 +16,35 @@ describe("Test calculateRank", () => {
     ).toStrictEqual({ level: "A+", score: 49.16605417270399 });
   });
 });
+
+describe("Test A++ calculateRank", () => {
+  it("should calculate rank correctly", () => {
+    expect(
+      calculateRank({
+        totalCommits: 1000,
+        totalRepos: 50,
+        followers: 1000,
+        contributions: 610,
+        stargazers: 4000,
+        prs: 3000,
+        issues: 2000,
+      })
+    ).toStrictEqual({ level: "A++", score: 32.89583911952869 });
+  });
+});
+
+describe("Test calculateRank for new account", () => {
+  it("should calculate rank correctly", () => {
+    expect(
+      calculateRank({
+        totalCommits: 0,
+        totalRepos: 0,
+        followers: 0,
+        contributions: 0,
+        stargazers: 0,
+        prs: 0,
+        issues: 0,
+      })
+    ).toStrictEqual({ level: "A+", score: 51.03597635931825 });
```

Comment:
I have seen that there are other ranks like B+ and C. This test means that the lowest rank is A+ which was not intended, I think.

Reply:
True, looks like `calculateRank` is broken. Will wait for input from @anuraghazra

Reply:
Would it maybe make sense to propose the score to be "0" in this case, with the lowest possible rank (C)?